libs/sdk-ui-kit/src/Button/tests/Button.test.tsx

```diff
@@ -1,95 +1,98 @@
-// (C) 2007-2020 GoodData Corporation
+// (C) 2007-2022 GoodData Corporation
 import React from "react";
-import { mount } from "enzyme";
-
+import { waitFor, render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import "@testing-library/jest-dom/extend-expect";
 import { Button } from "../Button";
 import { IButtonProps } from "../typings";
 
 function renderButton(options: Partial<IButtonProps>) {
-    return mount(<Button {...options} />);
+    return render(<Button {...options} />);
 }
 
 describe("ReactButton", () => {
     describe("click on button", () => {
-        let onClick: IButtonProps["onClick"];
-        beforeEach(() => {
-            onClick = jest.fn();
-        });
-
-        it("should call onClick callback on click", () => {
-            const wrapper = renderButton({
+        it("should call onClick callback on click", async () => {
+            const onClick = jest.fn();
+            renderButton({
                 type: "primary",
                 disabled: false,
                 onClick,
             });
 
-            wrapper.find("button").simulate("click");
-            expect(onClick).toHaveBeenCalledTimes(1);
+            await userEvent.click(screen.getByRole("button"));
+            await waitFor(() => {
+                expect(onClick).toHaveBeenCalledTimes(1);
+            });
         });
 
-        it("should not call onClick callback on click when disabled", () => {
-            const wrapper = renderButton({
+        it("should not call onClick callback on click when disabled", async () => {
+            const onClick = jest.fn();
+            renderButton({
                 type: "primary",
                 disabled: true,
                 onClick,
             });
 
-            wrapper.find("button").simulate("click");
-            expect(onClick).not.toHaveBeenCalled();
+            await userEvent.click(screen.getByRole("button"));
+            await waitFor(() => {
+                expect(onClick).not.toHaveBeenCalled();
+            });
         });
     });
 
     describe("selenium class", () => {
         it("should have correct selenium class", () => {
-            const wrapper = renderButton({
+            renderButton({
```

Comment:
Perhaps we can get rid of this test block. I don't think it has value enough to be tested. What do you think?

Reply:
makes sense